doc/api/Wallet.md

```diff
@@ -351,6 +351,80 @@ outputs
 }
 ```
 
+### Examples
+
+#### Send to single address
+
+###### Example POST Request
+Use _amount_ and _destination_ parameters.
+```
+/wallet/siacoins?amount=1000000000000000000000000&destination=1234567890abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789ab
+```
+
+###### Expected Response Code
+```
+200 OK
+```
+
+###### Example Response Body
+```json
+{
+  "transactionids": [
+    "3918e4a4b4cee46b3e5b28b8a1cc41c064a6f6002d162d396f296c201e6edc13",
+    "18b85b7d20f8a87bdadacf11e135ad44db1d93efd0613d23116e8cf255502762"
+  ]
+}
+```
+
+
+#### Send to set of addresses
+Use _outputs_ parameter which is a URL encoded JSON string. _amount_ and _destination_ parameters must be empty.
+
+
+###### Example POST Request
+```
+/wallet/siacoins?outputs=[%7B%22unlockhash%22:%221234567890abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789ab%22,%22value%22:%221000000000000000000000000%22%7D,%7B%22unlockhash%22:%22abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789ab1234567890%22,%22value%22:%228000000000000000000000000%22%7D,%7B%22unlockhash%22:%22cdef0123456789abcdef0123456789abcdef0123456789ab1234567890abcdef0123456789ab%22,%22value%22:%2220000000000000000000000000%22%7D]
```

Comment:
do the `"` marks really get encoded like this? That's ugly

Reply:
Yes.

Reply:
The markdown needs to be human readable, I'd go ahead and write them as quotes. Any reasonable backend is going to sanitize it for you